plugins/catalog-backend-module-aws/README.md

```diff
@@ -4,7 +4,56 @@ This is an extension module to the plugin-catalog-backend plugin, providing an
 `AwsOrganizationCloudAccountProcessor` that can be used to ingest cloud accounts
 as `Resource` kind entities.
 
-## Getting started
+The plugin also supports discovering catalog entities located in an S3 Bucket. See the [Backstage documentation](https://backstage.io/docs/integrations/aws-s3/discovery) for more details.
 
-See [Backstage documentation](https://backstage.io/docs/integrations/aws-s3/discovery) for details on how to install
-and configure the plugin.
+## For Ingesting AWS accounts
```

Comment:
This is more or less the same as what's available at https://backstage.io/docs/integrations/aws-s3/discovery/ right? We've discussed this balance a bit back and forth, wanting to have self-contained READMEs but also avoid duplication. So out of curiosity, how come you want to add these instructions to the README as well?

Reply:
At other providers like the one for GitHub, we have additional pages for other providers like those for org data: https://backstage.io/docs/integrations/github/org

This one is not about org data and not a provider (maybe, it was never migrated to entity providers).

I wonder whether or whether not we want to have it more at one place for the plugin.

Reply:
PS: there are more processors (we should omit the deprecated ones replaced by a provider)

Reply:
sure, should I move the changes somewhere here?https://github.com/backstage/backstage/tree/master/docs/integrations/github

Reply:
Yep let's move it over to the docs. I think a new `aws` dir in https://github.com/backstage/backstage/blob/master/docs/integrations/ is the way to go tbh. Then we can think about whether we want to move in `aws-s3` into that in the future.